docs/reference/usage.md

```diff
@@ -21,6 +21,9 @@ A value of `1` indicates the feature is active; any other value disables the fla
 
 - `RATIFY_DYNAMIC_PLUGINS`: (disabled) Enables Ratify to download plugins at runtime from an OCI registry by setting `source` on the plugin config
 
+- `RATIFY_USE_REGO_POLICY`: (disabled) Enables Ratify to generate verification reports in the format designed for evaluation by OPA engine with Rego language. Users can either use the OPA engine embedded in Ratify or the one within Gatekeeper.
+- `RATIFY_PASSTHROUGH_MODE`: (disabled) Enables Ratify to offload the policy decision-making to Gatekeeper. Ratify verify API will just return verification reports without the decision. `RATIFY_USE_REGO_POLICY` must be enabled to use this feature.
```

Comment:
Can we elaborate on why REGO_policy must be true for passthrough mode? I m coming from the perspective of ci pipleines, if passthrough mode is chosen to generate verifier report, all local policies are irrelevant?

Reply:
Synced offline. I'll keep the implementation, but add a version to the verificationResult and add more docs explaining it.